index.bs

```diff
@@ -28,10 +30,17 @@ dictionary CSSStyleSheetInit {
 	boolean alternate = false;
 	boolean disabled = false;
 };
+
+partial interface DocumentOrShadowRoot {
+	attribute StyleSheetList moreStyleSheets;
```

Comment:
Need to bikeshed this name, it's so bad. ^_^

Reply:
I agree, any suggestions? `adoptedStyleSheets` from @calebdwilliams sounds good, I think

Reply:
Sounds fine to me.